tests/adiabatic_initial_conditions_chunk/screen-output

```diff
@@ -5,31 +5,31 @@ Number of degrees of freedom: 1,444 (882+121+441)
 *** Timestep 0:  t=0 seconds
    Solving temperature system... 0 iterations.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 26+0 iterations.
+   Solving Stokes system... 25+0 iterations.
```

Comment:
The test uses a chunk geometry and tangential velocity boundary conditions on all boundaries but the outer boundary. It is the same test as for which I posted the picture. So I do think the results should change. Or did you mean another test?